cogs/moderation.py

```diff
@@ -43,6 +66,65 @@ async def ban(self, ctx, member: discord.Member, *, reason="No Reason Stated."):
         await member.send(embed=embed1)
         await member.ban(reason=reason)
 
+    class ActionReason(commands.Converter):
+        async def convert(self, ctx, argument):
+            ret = f'{ctx.author} (ID: {ctx.author.id}): {argument}'
+
+            if len(ret) > 512:
+                reason_max = 512 - len(ret) + len(argument)
+                raise commands.BadArgument(f'Reason is too long ({len(argument)}/{reason_max})')
+            return ret
+
+
+    class MemberID(commands.Converter):
+        async def convert(self, ctx, argument):
+            try:
+                m = await commands.MemberConverter().convert(ctx, argument)
+            except commands.BadArgument:
+                try:
+                    member_id = int(argument, base=10)
+                    m = await resolve_member(ctx.guild, member_id)
+                except ValueError:
+                    raise commands.BadArgument(f"{argument} is not a valid member or member ID.") from None
+                except MemberNotFound:
+                    # hackban case
+                    return type('_Hackban', (), {'id': member_id, '__str__': lambda s: f'Member ID {s.id}'})()
+
+            if not can_execute_action(ctx, ctx.author, m):
+                raise commands.BadArgument('You cannot do this action on this user due to role hierarchy.')
+            return m
+
+    @commands.command()
+    @commands.guild_only()
+    @commands.bot_has_permissions(ban_members=True)
+    @commands.has_permissions(ban_members=True)
+    async def multiban(self, ctx, members: commands.Greedy[MemberID], *, reason: ActionReason = None):
+        """Bans multiple members from the server.
+        This only works through banning via ID.
+        In order for this to work, the bot must have Ban Member permissions.
+        To use this command you must have Ban Members permission.
+        """
+
+        if reason is None:
+            reason = f'Action done by {ctx.author} (ID: {ctx.author.id})'
+
+        total_members = len(members)
+        if total_members == 0:
+            return await ctx.send('Missing members to ban.')
```

Comment:
```suggestion
            return await ctx.send('No members to ban.')
```
This is a bit more grammatically correct.